sys/dev/mxge/if_mxge.c

```diff
@@ -133,6 +133,21 @@ static device_method_t mxge_methods[] =
   DEVMETHOD_END
 };
 
+struct pci_device_table mxge_devs[] = {
+	{PCI_DEV(MXGE_PCI_VENDOR_MYRICOM, MXGE_PCI_DEVICE_Z8E),
+	 PCI_REVID(MXGE_PCI_REV_Z8E), PCI_DESCR("Myri10G-PCIE-8A")},
+	{PCI_DEV(MXGE_PCI_VENDOR_MYRICOM, MXGE_PCI_DEVICE_Z8E),
+	 PCI_REVID(MXGE_PCI_REV_Z8ES), PCI_DESCR("Myri10G-PCIE-8B")},
+	{PCI_DEV(MXGE_PCI_VENDOR_MYRICOM, MXGE_PCI_DEVICE_Z8E_9),
+	 PCI_REVID(MXGE_PCI_REV_Z8E), PCI_DESCR("Myri10G-PCIE-8A")},
+	{PCI_DEV(MXGE_PCI_VENDOR_MYRICOM, MXGE_PCI_DEVICE_Z8E_9),
```

Comment:
this is also a duplicate.
There should be two entries. The Z8E -> "....-8A" and Z8E_9 -> "....-8B"

Reply:
The revision id is different.